### ISSUE TITLE
Save files gone and new authentication after envoirment variables changed

### ISSUE THREAD

**Issue by Vy-Ro** [wolveix/satisfactory-server#344]
Hi , i'm a total noob with this and i'm having this issue ,
i added the cariable of tick rato to set it to 60 instead of 30 and after a reboot the server acted like a new server asking the confirmation for authentication, doing it the same as before and the save was gone. I tested afer again , every time i change somethign in the variables the server act like a new one and loose the save file. Is this suppoused to happen?
And where should the save file i upload stored to?  The 2 main folders required for the container are empty. Also this is expected behavior?
Than you for any response.

**Comment by wolveix** [comment]
No, this is not expected behaviour. Please fill out the issue template, otherwise I can't help you.
